lib/ethpy/ethpy/base/transactions.py

```diff
@@ -324,7 +329,7 @@ async def async_wait_for_transaction_receipt(
     """
     try:
         with Timeout(timeout) as _timeout:
-            poll_latency = start_latency + random.uniform(0, 1)
+            poll_latency = start_latency
```

Comment:
fine with removing this initial uniform, not necessary

Reply:
done in 916c248a7d9a9f3f45aef2079e75da332b01a5f1